scripts/tests/chiptest/test_definition.py

```diff
@@ -158,6 +155,18 @@ def __updateSetUpCode(self):
             raise Exception("Unable to find QR code")
         self.setupCode = qrLine.group(1)
 
+    def __terminateProcess(self):
+        if self.process:
+            self.process.terminate()  # sends SIGTERM
+            try:
+                self.process.wait(10)
+            except subprocess.TimeoutExpired:
+                logging.debug('Subprocess did not terminated on SIGTERM, killing it now')
```

Comment:
"did not terminate", not "did not terminated", @Apollon77

Reply:
Uups. Right typo. Will provide a fix PR later.

Reply:
@bzbarsky-apple see https://github.com/project-chip/connectedhomeip/pull/31702

Reply:
Thank you!